### PR TITLE
[Email] add action in manifest

### PR DESCRIPTION
**Pull request by KayMKM**
### Purpose
*What is the context of this pull request? Why is it being done?*
add action in manifest

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*
handle these actions in code

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json [hunk 1]
```diff
@@ -26,6 +26,117 @@
...
+          "type": "string",
+          "description": "Content of the email"
+        },
+        "sender": {
+          "type": "string",
+          "description": "Email address of email sender"
+        },
+        "reciever": {
+          "type": "string",
+          "description": "Semicolon separated list of email addresses to send"
```
Comment (xieofxie) [inline skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json]: If assume we have array, we could use array here

Comment (KayMKM) [reply skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json]: sure. fixed

File: skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json [hunk 2]
```diff
@@ -26,6 +26,108 @@
...
+      "description": "Delete an Email",
+      "type": "event",
+      "name": "DeleteEmail",
+      "value": {},
+      "resultValue": {
+        "$ref": "#/definitions/actionResult"
+      }
+    },
+    "replyEmail": {
+      "description": "Reply an Email",
```
Comment (darrenj) [inline skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json]: Reply to an email

File: skills/csharp/emailskill/wwwroot/manifest/manifest-1.1.json [hunk 3]
```diff
@@ -95,6 +95,108 @@
...
+      "description": "Delete an Email",
+      "type": "event",
+      "name": "DeleteEmail",
+      "value": {},
+      "resultValue": {
+        "$ref": "#/definitions/actionResult"
+      }
+    },
+    "replyEmail": {
+      "description": "Reply an Email",
```
Comment (darrenj) [inline skills/csharp/emailskill/wwwroot/manifest/manifest-1.1.json]: Reply to an email

File: skills/csharp/emailskill/wwwroot/manifest/manifest-1.1.json [hunk 4]
```diff
@@ -95,6 +95,108 @@
...
+          "type": "string",
+          "description": "Content of the email"
+        },
+        "sender": {
+          "type": "string",
+          "description": "Email address of email sender"
+        },
+        "reciever": {
+          "type": "array",
+          "description": "List of email addresses to forward",
```
Comment (darrenj) [inline skills/csharp/emailskill/wwwroot/manifest/manifest-1.1.json]: List of email addresses to send to

File: skills/csharp/emailskill/wwwroot/manifest/manifest-1.1.json [hunk 5]
```diff
@@ -95,6 +95,108 @@
...
+        },
+        "content": {
+          "type": "string",
+          "description": "Content of the email"
+        },
+        "sender": {
+          "type": "string",
+          "description": "Email address of email sender"
+        },
+        "reciever": {
```
Comment (darrenj) [inline skills/csharp/emailskill/wwwroot/manifest/manifest-1.1.json]: recipients

File: skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json [hunk 6]
```diff
@@ -26,6 +26,108 @@
...
+          }
+        }
+      }
+    },
+    "replyEmailInfo": {
+      "type": "object",
+      "properties": {
+        "replyMessage": {
+          "type": "string",
+          "description": "The message of reply"
```
Comment (darrenj) [inline skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json]: The reply message

File: skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json [hunk 7]
```diff
@@ -26,6 +26,108 @@
...
+        },
+        "content": {
+          "type": "string",
+          "description": "Content of the email"
+        },
+        "sender": {
+          "type": "string",
+          "description": "Email address of email sender"
+        },
+        "reciever": {
```
Comment (darrenj) [inline skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json]: recipients

File: skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json [hunk 8]
```diff
@@ -26,6 +26,108 @@
...
+        "replyMessage": {
+          "type": "string",
+          "description": "The message of reply"
+        }
+      }
+    },
+    "forwardEmailInfo": {
+      "type": "object",
+      "properties": {
+        "forwardReciever": {
```
Comment (darrenj) [inline skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json]: recipients

File: skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json [hunk 9]
```diff
@@ -26,6 +26,108 @@
...
+          "type": "array",
+          "description": "List of email addresses to forward",
```
Comment (darrenj) [inline skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json]: List of email addresses to forward to

File: skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json [hunk 10]
```diff
@@ -26,6 +26,108 @@
...
+          "type": "string",
+          "description": "Content of the email"
+        },
+        "sender": {
+          "type": "string",
+          "description": "Email address of email sender"
+        },
+        "reciever": {
+          "type": "array",
+          "description": "List of email addresses to send",
```
Comment (darrenj) [inline skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json]: List of email addresses to send to

File: skills/csharp/emailskill/wwwroot/manifest/manifest-1.1.json [hunk 11]
```diff
@@ -95,6 +95,108 @@
...
+        "replyMessage": {
+          "type": "string",
+          "description": "The message of reply"
+        }
+      }
+    },
+    "forwardEmailInfo": {
+      "type": "object",
+      "properties": {
+        "forwardReciever": {
```
Comment (darrenj) [inline skills/csharp/emailskill/wwwroot/manifest/manifest-1.1.json]: recipients

File: skills/csharp/emailskill/wwwroot/manifest/manifest-1.1.json [hunk 12]
```diff
@@ -95,6 +95,108 @@
...
+          "type": "array",
+          "description": "List of email addresses to forward",
```
Comment (darrenj) [inline skills/csharp/emailskill/wwwroot/manifest/manifest-1.1.json]: List of email addresses to forward to

File: skills/csharp/emailskill/wwwroot/manifest/manifest-1.1.json [hunk 13]
```diff
@@ -95,6 +95,108 @@
...
+          "items": {
+            "type": "string"
+          }
+        },
+        "forwardMessage": {
+          "type": "string",
+          "description": "The message of forward"
```
Comment (darrenj) [inline skills/csharp/emailskill/wwwroot/manifest/manifest-1.1.json]: The message to forward

**Review by darrenj — CHANGES_REQUESTED** [review]

File: skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json (hunk 2 above)
Comment (KayMKM) [reply skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json]: fixed

**Review by lzc850612 — APPROVED** [review]

**Pull request merged**